packages/runtime/src/runtime.ts

```diff
@@ -259,6 +281,36 @@ export class Runtime {
     }
   }
 
+  /**
+   * Creates Asset in Runtime
+   * @param name ASA name
+   * @param flags ASA Deployment Flags
+   */
+  createAsset (name: string, flags: ASADeploymentFlags): number {
+    const sender = flags.creator;
+    const senderAcc = this.assertAccountDefined(sender.addr, this.store.accounts.get(sender.addr));
+
+    // create asset
+    senderAcc.createAsset(++this.assetCounter, name, this.loadedAssetsDefs[name], sender.addr);
+    this.store.assetDefs.set(this.assetCounter, sender.addr);
+    return this.assetCounter;
+  }
```

Comment:
I see that we're not creating the assetCreation tx from sdk (`makeAssetCreateTxn(..)` from `algosdk`). I realize that we don't need to set ctx here, as we're not executing TEAL code. But the SDK constructor validates the `txParams (flags, payflags, asaDefs, addresses..)`, so we should either cover these validations or use the `sdk tx constructor`. Ignore if validations are already covered.

Reply:
if payFlags are validated then we can add it, only asaDefs are covered.